R/musMusculus/mouseAtlas.r

```diff
@@ -7,7 +7,7 @@ library(digest)
 library(here)
 
 # figure out project root
-proj_dir <- here()
+proj_dir <- here("Reference-Matrix-Generation")
```

Comment:
Sid, does running `here()` without arguments give the path to your ""Reference-Matrix-Generation" directory?

Reply:
No it gives just gives the path to my home directory.